spec/aws/rails/middleware/ebs_sqs_active_job_middleware_spec.rb

```diff
@@ -4,193 +4,287 @@
 require_relative 'elastic_beanstalk_job'
 require_relative 'elastic_beanstalk_periodic_task'
 
-module Aws
-  module Rails
-    describe EbsSqsActiveJobMiddleware do
-      # Simple mock Rack app that always returns 200
-      let(:mock_rack_app) { ->(_) { [200, { 'Content-Type' => 'text/plain' }, ['OK']] } }
+describe Aws::Rails::EbsSqsActiveJobMiddleware do
```

Comment:
massively re-did the test suite, let me know if I missed any cases 🤯 or it could be simplified